hieroplatform/src/main/java/org/hiero/utils/Randomness.java

```diff
@@ -0,0 +1,34 @@
+package org.hiero.utils;
+
+import org.apache.commons.math3.random.MersenneTwister;
+
+public class Randomness {
```

Comment:
It is not clear to me why write an interface. Do you imagine another IRandomness class in the project? An interface cannot include the 'getInstance' 'createInstance' functions. I think it makes sense (though not compulsory) that the PSG is a singleton. So the users will have to call specifically Randomness, and not refer to the Interface.